### ISSUE TITLE
4月版本更新后，不能修改人物的物品，希望更新，已经点星号

### ISSUE THREAD

**Issue by QAz234567** [magicbear/palworld-server-toolkit#110]
4月版本更新后，不能修改人物的物品，希望更新，已经点星号

**Comment by Banxiaxiala** [comment]
重新编译即可












ㅤ
***@***.***






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年4月9日(周二) 晚上7:11
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [magicbear/palworld-server-toolkit] 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 (Issue #110)




 
4月版本更新后，不能修改人物的物品，希望更新，已经点星号

**Comment by okasas100110** [comment]
> 重新编译即可 ㅤ ***@***.***
> […](#)
> ---原始邮件--- 发件人: ***@***.***&gt; 发送时间: 2024年4月9日(周二) 晚上7:11 收件人: ***@***.***&gt;; 抄送: ***@***.***&gt;; 主题: [magicbear/palworld-server-toolkit] 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 (Issue #110) 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

您好，请问重新编译的流程是什么，我尝试直接 python CxFreezeCompile.py build，但是好像缺少 save_tools 以及 pal_edit 的源码

**Comment by Banxiaxiala** [comment]
缺少就去那边拿一份 直接git上面搜然后把下载的最新源码放到指定位置












ㅤ
***@***.***






---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年4月12日(周五) 晚上8:01
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [magicbear/palworld-server-toolkit] 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 (Issue #110)




  
重新编译即可 ㅤ @.***
 …
 ---原始邮件--- 发件人: @.&gt; 发送时间: 2024年4月9日(周二) 晚上7:11 收件人: @.&gt;; 抄送: @.&gt;; 主题: [magicbear/palworld-server-toolkit] 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 (Issue #110) 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: @.&gt;
  
您好，请问重新编译的流程是什么，我尝试直接 python CxFreezeCompile.py build，但是好像缺少 save_tools 以及 pal_edit 的源码

**Comment by Necrul** [comment]
> > 缺少就去那边拿一份 直接git上面搜然后把下载的最新源码放到指定位置 ㅤ _**@**_.***
> > […](#)
> > ---原始邮件--- 发件人: _**@**_._**> 发送时间: 2024年4月12日(周五) 晚上8:01 收件人: _**@**_.**_>; 抄送: _**@**_._**_**@**_.**_>; 主题: Re: [magicbear/palworld-server-toolkit] 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 (Issue #110) 重新编译即可 ㅤ @.*** … ---原始邮件--- 发件人: @.> 发送时间: 2024年4月9日(周二) 晚上7:11 收件人: @.>; 抄送: @.>; 主题: [magicbear/palworld-server-toolkit] 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 (Issue #110) 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: @.> 您好，请问重新编译的流程是什么，我尝试直接 python CxFreezeCompile.py build，但是好像缺少 save_tools 以及 pal_edit 的源码 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: _**@**_.***>
> 
> 好的，谢谢，我成功重新编译了。 以下是我的步骤，供各位参考：
> 
> 1. pip3 install palworld-server-toolkit
> 2. 把 https://github.com/cheahjs/palworld-save-tools 拷贝到根目录的 save_tools 文件夹下
> 3. 根目录下新建文件夹 pal_edit，把 https://github.com/EternalWraith/PalEdit 拷贝到 pal_edit 下
> 4. 在根目录下的 palworld_server_toolkit 新建文件夹 PalEdit，将 https://github.com/EternalWraith/PalEdit 中的palworld_pal_edit/resources 文件夹拷贝到 PalEdit 下
> 5. python CxFreezeCompile.py build
> 6. 运行 build/exe.win32/editor.exe
> 7. Enjoy it!

So you were able to get item editing to work again doing this? I haven't been successful.

**Comment by misaka10384** [comment]
> > > 缺少就去那边拿一份 直接git上面搜然后把下载的最新源码放到指定位置 ㅤ _**@**_.***
> > > […](#)
> > > ---原始邮件--- 发件人: _**@**_._**> 发送时间: 2024年4月12日(周五) 晚上8:01 收件人: _**@**_.**_>; 抄送: _**@**_._**_**@**_.**_>; 主题: Re: [magicbear/palworld-server-toolkit] 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 (Issue #110) 重新编译即可 ㅤ @.*** … ---原始邮件--- 发件人: @.> 发送时间: 2024年4月9日(周二) 晚上7:11 收件人: @.>; 抄送: @.>; 主题: [magicbear/palworld-server-toolkit] 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 (Issue #110) 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: @.> 您好，请问重新编译的流程是什么，我尝试直接 python CxFreezeCompile.py build，但是好像缺少 save_tools 以及 pal_edit 的源码 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: _**@**_.***>
> > 
> > 
> > 好的，谢谢，我成功重新编译了。 以下是我的步骤，供各位参考：
> > 
> > 1. pip3 install palworld-server-toolkit
> > 2. 把 https://github.com/cheahjs/palworld-save-tools 拷贝到根目录的 save_tools 文件夹下
> > 3. 根目录下新建文件夹 pal_edit，把 https://github.com/EternalWraith/PalEdit 拷贝到 pal_edit 下
> > 4. 在根目录下的 palworld_server_toolkit 新建文件夹 PalEdit，将 https://github.com/EternalWraith/PalEdit 中的palworld_pal_edit/resources 文件夹拷贝到 PalEdit 下
> > 5. python CxFreezeCompile.py build
> > 6. 运行 build/exe.win32/editor.exe
> > 7. Enjoy it!
> 
> So you were able to get item editing to work again doing this? I haven't been successful.

是的  我按照他的操作执行一遍后程序就可以正常打开最新版Pal的Level.sav文件了  如果你需要这个 留下你的邮箱 我可以打包发你一份

**Comment by misaka10384** [comment]
> So you were able to get item editing to work again doing this? I haven't been successful.

是的 我按照他的操作执行一遍后程序就可以正常打开最新版Pal的Level.sav文件了 如果你需要这个 留下你的邮箱 我可以打包发你一份

**Comment by dimwen** [comment]
> > So you were able to get item editing to work again doing this? I haven't been successful.
> 
> 是的 我按照他的操作执行一遍后程序就可以正常打开最新版Pal的Level.sav文件了 如果你需要这个 留下你的邮箱 我可以打包发你一份

我一直编译不成功，是否可以发份给我？
我的邮箱是26261301@qq.com，谢谢

**Comment by misaka10384** [comment]
> 我一直编译不成功，是否可以发份给我？ 我的邮箱是[26261301@qq.com](mailto:26261301@qq.com)，谢谢

发你了 查收  然后文件太大 转Google云盘发送了

**Comment by dimwen** [comment]
收到，感谢，但google云盘那，我申请了访问，麻烦你通过一下




￢_￢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "magicbear/palworld-server-toolkit"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月26日(星期五) 下午4:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [magicbear/palworld-server-toolkit] 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 (Issue #110)





  
我一直编译不成功，是否可以发份给我？ ***@***.***，谢谢
  
发你了 查收  然后文件太大 转Google云盘发送了

**Comment by misaka10384** [comment]
> 收到，感谢，但google云盘那，我申请了访问，麻烦你通过一下

好了  Google云盘居然还要给访问权限   后续我还是换个云盘好了 
（主要是文件超过25M gmail会主动上传到Google云盘....

**Comment by dimwen** [comment]
感谢，已下载




￢_￢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "magicbear/palworld-server-toolkit"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年7月26日(星期五) 下午4:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [magicbear/palworld-server-toolkit] 4月版本更新后，不能修改人物的物品，希望更新，已经点星号 (Issue #110)





  
我一直编译不成功，是否可以发份给我？ ***@***.***，谢谢
  
发你了 查收  然后文件太大 转Google云盘发送了
